lib/matplotlib/axes/_axes.py

```diff
@@ -7649,6 +7649,8 @@ def specgram(self, x, NFFT=None, Fs=None, Fc=None, detrend=None,
         Z = np.flipud(Z)
 
         if xextent is None:
+            if Fs is None:
```

Comment:
I'd move this up to where we redefine the other constants like `NFFT` and `Fc`.

Reply:
Sure! Thanks for the suggestion.